src/Middleware/Diagnostics.EntityFrameworkCore/src/DatabaseDeveloperPageExceptionFilter.cs

```diff
@@ -16,17 +16,33 @@
 #nullable enable
 namespace Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore
 {
+    /// <summary>
+    /// An <see cref="IDeveloperPageExceptionFilter"/> that captures database-related exceptions that can be resolved by using Entity Framework migrations.
+    /// When these exceptions occur, an HTML response with details about possible actions to resolve the issue is generated.
+    /// </summary>
+    /// <remarks>
+    /// This should only be enabled in the Development environment. 
+    /// </remarks>
     public sealed class DatabaseDeveloperPageExceptionFilter : IDeveloperPageExceptionFilter
     {
         private readonly ILogger _logger;
         private readonly DatabaseErrorPageOptions _options;
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="DatabaseDeveloperPageExceptionFilter"/>.
+        /// </summary>
+        /// <param name="logger">The <see cref="ILogger"/>.</param>
+        /// <param name="options">The <see cref="IOptions{DatabaseErrorPageOptions}"/>.</param>
         public DatabaseDeveloperPageExceptionFilter(ILogger<DatabaseDeveloperPageExceptionFilter> logger, IOptions<DatabaseErrorPageOptions> options)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _options = options?.Value ?? throw new ArgumentNullException(nameof(options));
         }
 
+        /// <summary>
+        /// Handle <see cref="DbException"/> errors and output an HTML response with additional details.
+        /// </summary>
+        /// <inheritdoc />
```

Comment:
What happens with inheritdoc and a summary? Should you just have inheritdoc?

Reply:
Seems to work:

![image](https://user-images.githubusercontent.com/54385/95626853-f0863500-0a2f-11eb-90da-fa52b099e281.png)

![image](https://user-images.githubusercontent.com/54385/95627637-5e7f2c00-0a31-11eb-960d-e6cd97d2fd11.png)


This was tough to verify, because it stays as `/// <inheritdoc />` inside the compiled dll.

![image](https://user-images.githubusercontent.com/54385/95626900-098ee600-0a30-11eb-98e6-b347aa5f2bef.png)

While investigating this I ran across the issue you filed (#26434) about inherited docs not showing up on docs.microsoft.com. Good catch.

Reply:
Yeah that gotta be fixed.

Reply:
But glad that works!